lib/manager/pip_requirements/extract.js

```diff
@@ -16,16 +16,31 @@ module.exports = {
   extractPackageFile,
 };
 
-function extractPackageFile(content) {
+function extractPackageFile(content, _, config) {
   logger.trace('pip_requirements.extractPackageFile()');
 
-  let registryUrls;
+  const registryUrls = [];
+  let has_index_url = false;
   content.split('\n').forEach(line => {
     if (line.startsWith('--index-url ')) {
       const registryUrl = line.substring('--index-url '.length).split(' ')[0];
-      registryUrls = [registryUrl];
+      registryUrls.push(registryUrl);
```

Comment:
index-url should always be first in the array before extra-index-url. Do we need to worry about the case where the ordering is reversed in `requirements.txt`?